array_analyzer/workflows/interpolation_wf.py

```diff
@@ -1,49 +1,55 @@
-# bchhun, {2020-04-02}
-
-from array_analyzer.extract.txt_parser import *
-from array_analyzer.extract.img_processing import *
-from array_analyzer.load.debug_images import *
-from array_analyzer.transform.property_filters import *
-import array_analyzer.transform.array_generation as array_gen
 import re
 import time
-from datetime import datetime
-import skimage.io as io
+import os
+import numpy as np
 import pandas as pd
+import skimage.io as io
+
+import array_analyzer.extract.image_parser as image_parser
+import array_analyzer.extract.txt_parser as txt_parser
+import array_analyzer.extract.img_processing as img_processing
+import array_analyzer.load.debug_plots as debug_plots
+import array_analyzer.extract.constants as c
+import array_analyzer.transform.array_generation as array_gen
+from array_analyzer.extract.metadata import MetaData
 
-SCENION_SPOT_DIST = 82
 
-def interp(input_folder_, output_folder_, method='interp', debug=False):
+def interp(input_folder_, output_folder_, debug=False):
 
-    xml = [f for f in os.listdir(input_folder_) if '.xml' in f]
-    if len(xml) > 1:
-        raise IOError("more than one .xml file found, aborting")
-    xml_path = input_folder_+os.sep+xml[0]
+    # xml = [f for f in os.listdir(input_folder_) if '.xml' in f]
+    # if len(xml) > 1:
+    #     raise IOError("more than one .xml file found, aborting")
+    # xml_path = input_folder_+os.sep+xml[0]
 
     # parsing .xml
-    fiduc, spots, repl, params = create_xml_dict(xml_path)
+    # fiduc, spots, repl, params = create_xml_dict(xml_path)
 
     # creating our arrays
-    spot_ids = create_array(params['rows'], params['columns'])
-    antigen_array = create_array(params['rows'], params['columns'])
+    # spot_ids = create_array(params['rows'], params['columns'])
+    # antigen_array = create_array(params['rows'], params['columns'])
 
     # adding .xml info to these arrays
-    spot_ids = populate_array_id(spot_ids, spots)
+    # spot_ids = populate_array_id(spot_ids, spots)
     # spot_ids = populate_array_fiduc(spot_ids, fiduc)
 
-    antigen_array = populate_array_antigen(antigen_array, spot_ids, repl)
+    # antigen_array = populate_array_antigen(antigen_array, spot_ids, repl)
```

Comment:
Do you intend to bring back the commented code in this PR or is it safe to delete?